static/app/utils/replays/replayReader.tsx

```diff
@@ -328,6 +331,22 @@ export default class ReplayReader {
 
   getSDKOptions = () => this._optionFrame;
 
+  /**
+   * Checks the replay to see if user has any canvas elements in their
+   * application. Needed to inform them that we now support canvas in replays.
+   */
+  hasCanvasElementInReplay = memoize(() => {
+    const sdkOptions = this.getSDKOptions();
+    // @ts-expect-error TODO: Update SDK
+    if (sdkOptions?.canvas) {
```

Comment:
Need to release new SDK before this will work

Reply:
This field name seems... idk what it means. What's a better name?

Reply:
Updated